src/main/java/DBot.java

```diff
@@ -0,0 +1,132 @@
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Hashtable;
+
+public class DBot {
+    private static boolean isOn;
+    private static List<Task> taskList;
+
+    public static void main(String[] args) {
+        isOn = true;
```

Comment:
Consider using more descriptive names, that is easily identified to serve a definite purpose